### PR TITLE
HTTP example review and update

### PR DESCRIPTION
**Pull request by Pia007**

### CODE REVIEW

**Review by ianick87 — APPROVED** [review]

File: content/docs/examples/httpendpoints.md [hunk 1]
```diff
@@ -11,36 +11,116 @@
...
+## Integrate GET route with a Serverless Function
+Now, intergrate your route with a [Serverless Function](/docs/examples/functions/) by adding `lines 10 - 35`.
+{{<codesnippet `/source_code/http_examples/simple_api_example_get_serverless.py`>}}
 
 
 {{<break 1>}}
-## Integrate GET route with a Serverless function
-{{<codesnippet `/source_code/http_examples/simple_api_example_integration.py`>}}
+### Register your GET route
+Open the `src/cdev_project.py` file and register your `GET` route by replacing `lines 15-17` with the following:
```
Comment (DanielSanchez97) [inline content/docs/examples/httpendpoints.md:41]: I think we can remove this part. This part should be setup from the `quick-start` template.

File: content/docs/examples/httpendpoints.md [hunk 2]
```diff
@@ -11,36 +11,116 @@
...
+            "Content-Type": "application/json"
+        }
+    }
+```
+
+The `src/hello_world/resources.py` now looks like:
+{{<codesnippet `/source_code/http_examples/simple_api_example_post_serverless.py`>}}
+
+### Register your POST route
+Open `src/cdev_project.py` file and register your `POST` route by replacing `lines 15-17` with the following:
```
Comment (DanielSanchez97) [inline content/docs/examples/httpendpoints.md:107]: Again see above. I think we can remove this part. This part should be setup from the quick-start template.

**Review by DanielSanchez97 — CHANGES_REQUESTED** [review]
Small change to remove two sections then LGTM.

File: content/docs/examples/httpendpoints.md (hunk 1 above)
Comment (Pia007) [reply content/docs/examples/httpendpoints.md]: Thanks Daniel.  I have removed that section from the Get and Post route examples.

File: content/docs/examples/httpendpoints.md [hunk 3]
```diff
@@ -11,36 +11,98 @@
...
+
+Open the `src/hello_world/resources.py` file and start by importing the following resources:
+{{<codesnippet `/source_code/http_examples/simple_api_example_resources.py`>}}
 
 
 {{<break 1>}}
-## Create routes
-{{<codesnippet `/source_code/http_examples/simple_api_example_get.py`>}}
+## Create a route
+Next, add lines `lines 8, 10 and 12` to define your project, `API` and create your route.
```
Comment (jtfrench23) [inline content/docs/examples/httpendpoints.md:29]: Need to remove one of the "lines" in line 29

**Review by jtfrench23 — CHANGES_REQUESTED** [review]
I just found one grammar fix

File: content/docs/examples/httpendpoints.md (hunk 3 above)
Comment (Pia007) [reply content/docs/examples/httpendpoints.md]: Good catch!

**Review by jtfrench23 — APPROVED** [review]

**Review by DanielSanchez97 — APPROVED** [review]
LGTM

**Pull request merged**